data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		event "returned geocoris" 365
+		conversation
+			`You have successfully rescued the kidnapped Hai and reclaimed the Geocoris. Something is different about it...`
+				decline
+
+event "returned geocoris"
+		
+ship "Geodesic Geocoris"
+	sprite "ship/geodesic geocoris"
+	thumbnail "thumbnail/geodesic geocoris"
+	attributes
+		category "Heavy Warship"
+		"cost" 23540000
+		"shields" 26600
+		"hull" 7100
+		"required crew" 27
+		"bunks" 48
+		"mass" 955
+		"drag" 21.3
+		"heat dissipation" 0.4
+		"fuel capacity" 800
+		"cargo space" 1003
+		"outfit space" 555
+		"weapon capacity" 250
+		"engine capacity" 134
+		"energy capacity" 5200
+		"shield generation" 6.0
+		"shield energy" 8.8
+		"hull repair rate" 0.8
+		"hull energy" 0.8
+		"hull heat" 1.2
+		weapon
+			"blast radius" 400
+			"shield damage" 4000
+			"hull damage" 2000
+			"hit force" 6000
+	outfits
+		"Chameleon Anti-Missile" 2
+		"Quad Blaster Turret" 2
+		"Pulse Turret" 2
+		
+		"Quantum Keystone"
+		"Boulder Reactor" 2
+		"Liquid Helium Cooler"
+		"Security Station" 2
+		"Pulse Rifle" 30
+
+		"A525 Atomic Steering"
+		`"Biroo" Atomic Thruster`
+		"AR120 Reverse Thruster" 2
+		Hyperdrive
+
+	turret 0 -82.5 "Pulse Turret"
+	turret -95 -17 "Chameleon Anti-Missile"
+	turret 95 -17 "Chameleon Anti-Missile"
+	turret -29 3 "Quad Blaster Turret"
+	turret 29 3 "Quad Blaster Turret"
+	turret 0 91.5 "Pulse Turret"
+	engine -95 41.5 0.9
+	engine 95 41.5 0.9
+	engine 1 154.5 0.9
+	leak "leak" 40 50
+	leak "flame" 40 80
+	leak "big leak" 70 30
+	explode "tiny explosion" 50
+	explode "small explosion" 50
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "This modified Geocoris has identical cargo capacity, but is modeled off one that was the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings, and a crucial element of the ship's system core is not manufactured, but grown from initial samples taken from the first occurrence. This means that some systems are built-in and unable to be removed, and also that it is substantially more expensive than its regular version. It is, however, significantly more survivable."
+	
+ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
+	outfits
+		"Electron Turret" 3
+		"Chameleon Anti-Missile" 2
+		"AR120 Reverse Thruster" 2
+		"Boulder Reactor"
+		"Hai Ravine Batteries"
+		"Outfits Expansion"
+		"Large Heat Shunt"
+		"Quantum Keystone"
+		"Pulse Rifle" 14
+		`"Benga" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Jump Drive"
+		Hyperdrive
+		"Failed Control System"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Chameleon Anti-Missile"
+	turret "Chameleon Anti-Missile"
+	turret "Electron Turret"
+
+mission "Hai Leaks Response 5"
```

Comment:
Another instance where I'd prefer an on offer conversation to an on complete from the previous mission.